src/Server.php

```diff
@@ -153,7 +177,7 @@ public function setClass($class, $namespace = '', $argv = null)
      * @param  mixed $data
      * @return Error
      */
-    public function fault($fault = null, $code = 404, $data = null)
+    public function fault($fault = null, $code = 404, $data = null): Error
```

Comment:
@Xerkus We can't because this is implementation of abstract/interface from laminas-server.... Unless we update first laminas-server, what I've suggested.

Reply:
As @michalbundyra mentioned, argument type declaration is not compatible with super. Since I will only be available until the end of next week and then be gone for two months, I would prefer completing this PR first.